File: Modul 4/C/PRAK403.c
#include <stdio.h>

int main() {
    printf("Input\n");
    int angka1, angka2;
    scanf("%d %d", &angka1, &angka2);
    int a = angka1;
    int b = angka2;
    printf("Output\n");

    if (angka1 < angka2) {
        for (int i = angka1; i <= angka2; i++, a++, b--) {
            printf("%d %d", a, b);
            if (a == angka2) {
                break;
            } else {
                printf(" - "); }
        }
    } else {
        for (int i = angka2; i <= angka1; i++, a--, b++) {
            printf("%d %d", a, b);
            if (a == angka2) {
                break;
            } else {
                printf(" - "); }
        }
    }
    return 0;
}

File: Modul 4/PYTHON/PRAK403.py
print("Input")
angka= input().split() 
angka1, angka2 = map(int, angka)
x = int(angka1)
y = int(angka2)
print("Output")
if angka1 < angka2:
    for i in range(int(angka1), int(angka2)+1):
        print(x, end=' ')
        print(y, end=' ')
        x += 1
        y -= 1
        if y == int(angka1) - 1:
            break
        else:
            print("-", end=' ')
else:
    for i in range(int(angka2), int(angka1)+1):
        print(x, end=' ')
        print(y, end=' ')
        x -= 1
        y += 1
        if x == int(angka2) - 1:
            break
        else:
            print("-", end=' ')